app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
```

Comment:
Using `self.people[self.name] = self` assumes that `people` is an instance attribute, but it is actually a class attribute. This can lead to shared state across instances, which is likely not the intended behavior.